contracts/land/LANDEvolution.sol

```diff
@@ -2,16 +2,10 @@ pragma solidity ^0.4.24;
 
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import "openzeppelin-solidity/contracts/token/ERC721/ERC721Token.sol";
+import "./LANDbase.sol";
 
 
-// TODO:
-contract LANDEvolution is ERC721Token("Evolution Land Oasis","EVOLAND"), Ownable{
-
-    uint256 constant clearLow = 0xffffffffffffffffffffffffffffffff00000000000000000000000000000000;
-    uint256 constant clearHigh = 0x00000000000000000000000000000000ffffffffffffffffffffffffffffffff;
-    uint256 constant factor = 0x100000000000000000000000000000000;
-
-
+contract LANDEvolution is ERC721Token("OASIS","EVL"), Ownable, LANDbase{
```

Comment:
Contract Name coding style:

http://solidity.readthedocs.io/en/v0.4.24/style-guide.html#contract-and-library-names